packages/twenty-front/src/modules/page-layout/widgets/graph/graphWidgetBarChart/components/CustomBarItem.tsx

```diff
@@ -87,10 +88,20 @@ export const CustomBarItem = <D extends BarDatum>({
   );
   const handleMouseEnter = useCallback(
     (event: MouseEvent<SVGRectElement>) => {
-      onMouseEnter?.(barData, event);
+      const barDatumWithGeometry = {
+        ...barData,
+        barElement: event.currentTarget,
+        barX: bar.x,
+        barY: bar.y,
+        barWidth: bar.width,
+        barHeight: bar.height,
+        barAbsoluteX: bar.absX,
+        barAbsoluteY: bar.absY,
+      } as BarDatumWithGeometry;
```

Comment:
prefere either satisfy or barDatumWithGeometry: BarDatumWithGeometry because as is less safe